src/drupal/application/overlay/docroot/sites/default/settings.local.php.twig

```diff
@@ -15,6 +15,11 @@ $databases['default']['default'] = array (
   'port'      => '3306',
   'namespace' => 'Drupal\\Core\\Database\\Driver\\mysql',
   'driver'    => 'mysql',
+{% if version_compare(@('drupal.major_version'), '10', '>=') %}
+  'init_commands' => [
+    'isolation_level' => 'SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED',
+  ],
+{% endif %}
```

Comment:
note also this might get wiped unless `$databases['default']['default'] = ...` is removed from settings.php, as it's defined after settings.local.php is included